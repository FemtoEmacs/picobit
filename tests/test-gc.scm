(define (foo c)
  (let ((x (cons 2 3)))
    (cons 3 (foo c)))
  ;; (if (display (car (cons 2 3)))
;;       (foo)
;;       (foo))
  )
(foo '())
