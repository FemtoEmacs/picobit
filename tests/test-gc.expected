:0080000080
:10800000FBD7000002B582EED8010607CD2102CDD4
:0B801000B4001471010722B570CDEF21
:00000001FF
